packages/openfdd-agent-shell/src/openfdd_agent_shell/prompts.py

```
@@ -38,12 +39,105 @@ def build_system_prompt(manifest: Manifest) -> str:
+    memory = MemoryStore(manifest)
+    open_count = memory.count_open_divergence_entries()
+    paths = MemoryPaths.from_manifest(manifest)
+    arch_log = _repo_rel(manifest, paths.divergence_file)
+    checkpoints = _repo_rel(manifest, manifest.wake.checkpoints_file)
+    read_block = "\n".join(f"- {item}" for item in _wake_read_list(manifest))
+    return (
+        f"Scheduled Open-FDD wake (critique after up to {mini_count} mini runs).\n\n"
+        f"Read:\n{read_block}\n\n"
+        "Tasks:\n"
+        "1) Summarize what likely changed this wake (BUILD_CHECKPOINTS Done recently, daily notes, workspace diffs, cron run logs).\n"
+        f"2) Rewrite {checkpoints}: **Last critique**, **Current sprint**, and replace **Next for mini** with 3–8 concrete tasks for the next wake.\n"
```

Ruff (0.15.12)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)